defaults/main.yml

```diff
@@ -43,3 +43,8 @@ nats_cli_version: '0.0.35'
 nats_exporter_enabled: false
 nats_exporter_version: '0.10.1'
 nats_exporter_options: '-port 7777 -channelz -connz -routez -serverz -subz -varz'
+
+nats_jetstream_enabled: false
+nats_jetstream_storage_dir: '/var/lib/nats/data'
```

Comment:
this directory should be created beforehand, otherwise nats server failed to start:
```
[629] 2023/11/21 06:55:46.394797 [INF] Starting JetStream
[629] 2023/11/21 06:55:46.395082 [FTL] Can't start JetStream: could not create storage directory - mkdir /var/lib/nats: permission denied
```